charts/buildbuddy-executor/templates/deployment.yaml

```diff
@@ -56,7 +56,8 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.imagePullPolicy }}
           {{- if .Values.customExecutorCommand }}
-          command: {{ .Values.customExecutorCommand }}
+          command:
+          {{- .Values.customExecutorCommand | toYaml | nindent 10}}
```

Comment:
@JamesMBartlett How does your desired command look like?

given a config like this

```yaml
customExecutorCommand: ["foo", "bar"]
```

Using `helm template buildbuddy-executor charts/buildbuddy-executor`

I got this before this PR
```yaml
....
        - name: buildbuddy-executor
          image: "gcr.io/flame-public/buildbuddy-executor-enterprise:enterprise-v2.12.43"
          imagePullPolicy: IfNotPresent
          command: [foo bar]
          resources:
...
```

and this after this PR

```yaml
...
        - name: buildbuddy-executor
          image: "gcr.io/flame-public/buildbuddy-executor-enterprise:enterprise-v2.12.43"
          imagePullPolicy: IfNotPresent
          command:
          - foo
          - bar
          resources:
...
```

Both seem valid but the earlier is more in line with the examples in https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/.

WDYT?

Reply:
The former is missing the comma, so its not valid.

Reply:
Sorry I missed the comma part when reviewing the original PR.

This makes sense